utils/scandetails.go

```diff
@@ -142,6 +142,11 @@ func (sc *ScanDetails) RunInstallAndAudit(workDirs ...string) (auditResults *aud
 }
 
 func (sc *ScanDetails) runInstallIfNeeded(workDir string) (err error) {
+	// Temporary block until all 'install' logic moves out from frogbot and this code will be deprecated
+	if sc.InstallCommandName == "npm" {
+		return nil
+	}
```

Comment:
I assume you'd like to remove this code before mergint the PR.

Reply:
this will be removed at the end after deprecating the run of install command from Yarn, NuGet and Dotnet as well

Reply:
👍